c/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+cmake_minimum_required(VERSION 3.9)
+
+project(libblake3 VERSION 0.1.0 DESCRIPTION "BLAKE3 C implementation")
+enable_language(C ASM)
+
+include(GNUInstallDirs)
+
+set(blake3_SOURCES)
+list(APPEND blake3_SOURCES
+  blake3.c
+  blake3_dispatch.c
+  blake3_portable.c)
+
+if(CMAKE_SYSTEM_PROCESSOR STREQUAL x86_64)
+  list(APPEND blake3_SOURCES
+    blake3_avx2_x86-64_unix.S
+    blake3_avx512_x86-64_unix.S
+    blake3_sse41_x86-64_unix.S)
```

Comment:
Let's add SSE2 to this list. I think SSE2 support might've been added to the repo after this original CMake file was written. Something like this:

```diff
diff --git a/c/CMakeLists.txt b/c/CMakeLists.txt
index 6675709..e00573a 100644
--- a/c/CMakeLists.txt
+++ b/c/CMakeLists.txt
@@ -15,14 +15,17 @@ if(CMAKE_SYSTEM_PROCESSOR STREQUAL x86_64)
   list(APPEND blake3_SOURCES
     blake3_avx2_x86-64_unix.S
     blake3_avx512_x86-64_unix.S
+    blake3_sse2_x86-64_unix.S
     blake3_sse41_x86-64_unix.S)
 elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL i686)
   list(APPEND blake3_SOURCES
     blake3_avx2.c
     blake3_avx512.c
+    blake3_sse2.c
     blake3_sse41.c)
   set_source_files_properties(blake3_avx2.c PROPERTIES COMPILE_FLAGS -mavx2)
   set_source_files_properties(blake3_avx512.c PROPERTIES COMPILE_FLAGS "-mavx512f -mavx512vl")
+  set_source_files_properties(blake3_sse2.c PROPERTIES COMPILE_FLAGS -msse2)
   set_source_files_properties(blake3_sse41.c PROPERTIES COMPILE_FLAGS -msse4.1)
 elseif((ANDROID_ABI STREQUAL armeabi-v7a) OR
        (CMAKE_SYSTEM_PROCESSOR STREQUAL aarch64) OR (CMAKE_SYSTEM_PROCESSOR STREQUAL arm64))
```

Reply:
Right so i'll add that right now